src/index.css

```
@@ -1,41 +1,81 @@
+/* --- General Reset and Box-Sizing --- */
+
+/* Ensure consistent box-sizing across all elements */
+*, *::before, *::after { box-sizing: border-box; /* Prevents padding and borders from affecting element size */ }
+
+/* Reset margin and padding for all elements */
+* {
+	margin: 0;
+	padding: 0; /* Removes default spacing */
+}
+
+/* --- Remove Default Styling for Lists --- */
+ul, ol { list-style: none; /* Removes default bullet points and numbering */ }
```

```suggestion
nav :is(ul, ol) { list-style: none; /* Removes default bullet points and numbering */ }
```

We _**DON'T**_ want to remove list styles from everything. The only place where that is consistently done and makes sense is when a list is used as part of a navigation menu.

Like this:
```html
<nav>
  <ul>
    <li><a href="#">Link 1</a></li>
    <li><a href="#">Link 2</a></li>
    <li><a href="#">Link 3</a></li>
  </ul>
</nav>
```

For other common scenarios, like in a text body, we want to keep the list styles.